CoopANDlastfm.html

```diff
@@ -7,25 +7,16 @@
 		<title>Sound and color with me</title>
 		<link rel="stylesheet" type="text/css" href="CoopANDlastfm.css"/>
 		<link rel="stylesheet" type="text/css" href="http://code.jquery.com/ui/1.9.2/themes/base/jquery-ui.css"/>
-		
-		  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.5.11/p5.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.5.11/addons/p5.dom.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.5.11/addons/p5.sound.min.js"></script>
-		
+		<link rel="stylesheet" href="//ajax.googleapis.com/ajax/libs/jqueryui/1.11.3/themes/smoothness/jquery-ui.css">
 	</head>
 	<body>
 		<h2>SOUND AND COLOR</h2>
 		<h3> a project in synesthesia</h3>
 		<input id="query" type="text" placeholder="ENTER A COLOR"></input>
 		<button class="aButton" id="search">SUBMIT</button>
 		<div id="images"></div>
-		<div id="searchResults" div class="ui-widget-content"> </div>
```

Comment:
The extra div in this element might have been a reason you were having problems on the reload